app/assets/javascripts/oxalis/view/right-menu/dataset_info_tab_view.js

```diff
@@ -84,12 +84,11 @@ const shortcuts = [
 ];
 
 export function calculateZoomLevel(flycam: Flycam, dataset: APIDataset): number {
-  const zoom = getPlaneScalingFactor(flycam);
+  // todo
```

Comment:
What is this todo about?

Reply:
Hmm, good point, I forgot about that. The viewport extent in nm is not shown right now (since it differs per viewport edge). Will have to find a solution for that, yet.